Package.swift

```diff
@@ -4,8 +4,8 @@ import PackageDescription
 let package = Package(
     name: "FluidAudio",
     platforms: [
-        .macOS(.v13),
-        .iOS(.v16),
+        .macOS(.v14),
```

Comment:
our min version for models are compiled against this so no point of supporting older versions. only makes it more confusing.